src/dbnode/persist/fs/index_lookup.go

```diff
@@ -158,8 +160,10 @@ func newNearestIndexOffsetLookupFromSummariesFile(
 	decoderStream xmsgpack.ByteDecoderStream,
 	numEntries int,
 	forceMmapMemory bool,
+	reporterOptions mmap.ReporterOptions,
```

Comment:
Same here, might be better to just propagate the `mmap.Reporter` then set the `ReporterOptions` here ourselves?

That way it's pretty clear and in control of setting up the `Context` itself, etc. Also not clear if it needs to add tags if needs to check if `Context` already has tags and to take a copy and modify, etc. Just seems cleaner to construct `ReporterOptions` and `Context` with the given reporter right before use.